<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="MovieListLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Movie list:                                                                                       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UndoButton">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RedoButton">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="MovieList">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="TitleLabel">
                <property name="text">
                 <string>Title:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="GenrLabel">
                <property name="text">
                 <string>Genre:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="YearLabel">
                <property name="text">
                 <string>Year:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LikesLabel">
                <property name="text">
                 <string>Likes:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="TrailerLabel">
                <property name="text">
                 <string>Trailer:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="TitleLine"/>
              </item>
              <item>
               <widget class="QLineEdit" name="GenreLine"/>
              </item>
              <item>
               <widget class="QLineEdit" name="YearLine"/>
              </item>
              <item>
               <widget class="QLineEdit" name="LikesLine"/>
              </item>
              <item>
               <widget class="QLineEdit" name="TrailerLine"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="AddButton">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="DeleteButton">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="UpdateButton">
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="UpdateTitleLabel">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Update title:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UpdateTitleLine"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Type anything to filter:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FilterLine"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
